CONTRIBUTING.md

```diff
@@ -120,3 +120,13 @@ cd ReSwift-gh-pages
 # Documentation is ready to be committed.
 ```
 
+## Pull Request Technicalities
+
+Here are some hard facts for reference:
```

Comment:
I would rephrase this to something that sounds a little friendlier :) Or maybe just remove this sentence and jump right into the bullet point list.